pkg/deploy/spindeploy/save.go

```diff
@@ -36,6 +36,10 @@ func (d *Deployer) deployConfig(ctx context.Context, scheme *runtime.Scheme, gen
 	// the status. But things happen.
 	d.log.Info(fmt.Sprintf("saving %d manifests across %d services", count, len(gen.Config)))
 	for k := range gen.Config {
+		if _, ok := unmanage[k]; ok {
+			logger.Info(fmt.Sprintf("skip applying updates for %s", k))
+			continue
+		}
 		s := gen.Config[k]
 		if s.Deployment != nil {
```

Comment:
it would skip only the Deployment for the given services adding the condition here